packages/hooks/src/useCloseDerivAccount.ts

```diff
@@ -0,0 +1,16 @@
+import { useRequest } from '@deriv/api';
+
+/**
+ * A custom hook that calls the `account_closure` api. This call allows clients to close all their accounts (including virtual-money account)
+ */
+const useCloseDerivAccount = () => {
+    const { data, mutate, error, ...rest } = useRequest('account_closure');
+    return {
+        data,
+        mutate,
+        error,
+        ...rest,
+    };
```

Comment:
is we're not doing any data mutation here we can return the response directly rather than destructuring
```suggestion
    const response = useRequest('account_closure');
    return response;
```